### ISSUE TITLE
Can I create two instances of the <Calendar.List /> with useDateRange hooks ?

### ISSUE THREAD

**Issue by ioeldev** [MarceloPrado/flash-calendar#51]
I have two screens with the Calendar.List but when one update the state of useDateRange it gets reflected on the second component, even though the Calendar.List have two different "key" props.
Is it possible to instantiate the useDateRange twice ?

**Comment by MarceloPrado** [comment]
The `useDateRange` hook is a plain React hook that exposes a useState's value + some setters. Did you try something like this?
```tsx
const YourApp = ({ currentScreen }) => {
  const dateRangePropsForFirstScreen = useDateRange()
  const dateRangePropsForSecondScreen = useDateRange()
 

  return currentScreen === 'one' ? 
    <ScreenOne {...dateRangePropsForFirstScreen} /> : 
    <ScreenTwo {...dateRangePropsForSecondScreen} />
}
```

**Comment by ioeldev** [comment]
Will try, it's just that i am initializing the useDateRange inside of each components so i thought i wouldn't interfere

**Comment by MarceloPrado** [comment]
A repro example would be helpful to understand what's going on

**Comment by MarceloPrado** [comment]
@ioeldev fixed in version `1.1.0`
